local-interchain/python/helpers/file_cache.py

```diff
@@ -33,6 +34,7 @@ def get_chain_start_time_from_logs() -> int:
     def get_cache_or_default(contracts: dict, ictest_chain_start: int) -> dict:
         cache_time: str | int = 0
         if os.path.exists(contracts_json_path):
+            os.makedirs(os.path.dirname(contracts_json_path), exist_ok=True)
```

Comment:
i don't Python so can't speak to these changes but i will assume these two lines are correct

Reply:
yes, just creates a nested directory if it does not exist. if it does, it does nothing